src/class-parsely.php

```diff
@@ -362,9 +362,10 @@ public function get_options() {
 		 *
 		 * @var Parsely_Options|null
 		 */
-		$options = get_option( self::OPTIONS_KEY, $this->option_defaults );
+		$options = get_option( self::OPTIONS_KEY, null );
 
 		if ( ! is_array( $options ) ) {
```

Comment:
One edge case where this can be a problem is if the DB query to lookup options fails. We've seen this happen on VIP a few times which can end up resetting the state.

But, it's an edge case and a lower-level issue so I'm not sure it's worth solving for here. If we did want to solve for it, we'd need to check for something like this:

```suggestion
		if ( ! is_array( $options ) && ! $wpdb->last_error ) {
```

Reply:
My research indicates that `$wpdb->last_error` is flushed before every query, so we don't risk getting an error that comes from a previous query. If we were to do this check, I'd probably go with something like this:

```php
global $wpdb

// If options aren't an array or a database error occurred, return the default options.
if ( ! is_array( $options ) || '' !== $wpdb_last_error ) {
```

Unsure this adds value though, as in case of a database reading failure, `$options` should be `false` or `null`, which will still return the default options.

Reply:
Sorry, what I meant was that we want to prevent the default options from being used when the db query fails. Otherwise, a temporary issue with the database would reset all Parsely options. 

But like I said, it's an edge case and also not a new problem, so i'm okay with skipping this for now.